README.md

```diff
@@ -60,7 +62,7 @@ Cohere Endpoint | Function
 To view an up-to-date list of available models please consult the [Cohere CLI](https://docs.cohere.ai/command/). To get started try out `baseline-shrimp` or `baseline-seal`.
 
 ## Responses
-All of the endpoint functions will return some Cohere object (e.g. for generation, it would be `Generation`). The responses can be found as instance variables of the object (e.g. for generation, it would be `Generation.text`). The names of these instance variables and a detailed breakdown of the response body can be found in the [Cohere Docs](https://docs.cohere.ai/).
+All of the endpoint functions will return some Cohere object (e.g. for generation, it would be `Generation`). The responses can be found as instance variables of the object (e.g. for generation, it would be `Generation.text`). The names of these instance variables and a detailed breakdown of the response body can be found in the [Cohere Docs](https://docs.cohere.ai/). Printing the Cohere response object itself will display an organized view of the instance variables.
```

Comment:
```suggestion
All of the endpoint functions will return a Cohere object corresponding to the endpoint (e.g. for generation, it would be `Generation`). The responses can be found as instance variables of the object (e.g. generation would be `Generation.text`). The names of these instance variables and a detailed breakdown of the response body can be found in the [Cohere Docs](https://docs.cohere.ai/). Printing the Cohere response object itself will display an organized view of the instance variables.
```